lib/client/checks/config/index.ts

```diff
@@ -86,3 +87,17 @@ const brokerClientVersionCheck = (config: Config): Check => {
     },
   } satisfies Check;
 };
+
+const brokerTokenCheck = (config: Config): Check => {
+  return {
+    id: 'broker-token-validation',
+    name: 'Broker Token Validation Check',
+    enabled: !config.universalBrokerEnabled,
+    check: async function (): Promise<CheckResult> {
+      return await validateBrokerClientVersionAgainstServer(
```

Comment:
```suggestion
      return await validateBrokerToken(
```

I guess it should be the different method here.

Reply:
Good catch. Not sure what happened there.